Model/Push.php

```diff
@@ -591,6 +591,12 @@ private function isPushNeeded()
             return false;
         }
 
+        $statusCodePendingProcessing = $this->helper->getStatusCode('BUCKAROO_MAGENTO2_STATUSCODE_PENDING_PROCESSING');
+        if ($this->hasPostData('brq_statuscode', $statusCodePendingProcessing)
```

Comment:
what exactly happens when the payment is in pending processing that causes the order to be cancelled?

Reply:
I believe the order status has changed, and when the next push comes, it cannot process the success push properly. I haven't tested the Payconiq payment method yet. I will wait to receive feedback to see if the fix helps.